.travis.yml

```diff
@@ -0,0 +1,27 @@
+sudo: required
+
+# Only run builds for PRs and commits into master
+branches:
+  only:
+    - master
+
+language: python
+python:
+ - 3.7
+dist: xenial
+
+env:
+  global:
+  - PIPENV_VENV_IN_PROJECT=1
+  - PIPENV_IGNORE_VIRTUALENVS=1
+
+install:
+ - pipenv install --dev --deploy && touch .installed
```

Comment:
we don't need pipenv. 
```pip install pre-commit==version --pre```

Reply:
We have way more dependencies than just pre-commit. See `Pipfile.lock`.